bundle/DependencyInjection/Configuration.php

```diff
@@ -57,5 +57,115 @@ protected function addConfiguration(ArrayNodeDefinition $rootNode): void
             ->booleanNode('render_missing_field_info')
                 ->info('Whether to render useful debug information in place of a missing field')
             ->end();
+
+        $keyValidator = static function ($v): bool {
+            foreach (\array_keys($v) as $key) {
+                if (!\is_string($key) || !\preg_match('/[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*$/A', $key)) {
+                    return true;
+                }
+            }
+
+            return false;
+        };
+
+        $idMapper = static function ($v) {
+            if (\is_int($v)) {
+                return ['id' => $v];
+            }
+
+            return ['remote_id' => $v];
+        };
+
+        /* @noinspection NullPointerExceptionInspection */
+        $systemNode
+            ->arrayNode('named_object')
```

Comment:
If we're gonna keep this one, maybe a better name would be in plural: `named_objects`?